### PR TITLE
feat(ui): add a menu to change the language in the dashboard

### PR DESCRIPTION
**Pull request by bimalgrg519**

### CODE REVIEW

File: ui/web-v2/apps/admin/src/components/Header/index.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { FC, memo } from 'react';
 
+import { LanguageSelect } from '../LanguageSelect/LanguageSelect';
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/components/Header/index.tsx:3]: For new or changes in the imports, please use the absolute path.

E.g.
```ts
import { LanguageSelect } from '@/components/LanguageSelect';
```

File: ui/web-v2/apps/admin/src/lang/index.ts [hunk 2]
```diff
@@ -1,15 +1,25 @@
 import { createIntl, createIntlCache } from 'react-intl';
 
+import en from '../assets/lang/en.json';
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/lang/index.ts:3]: Same here. Please use the absolute path.

File: ui/web-v2/apps/admin/src/components/LanguageSelect/LanguageSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  readonly label: string;
+  readonly value: LanguageTypes;
+}
+
+const languageList: ILanguageItem[] = [
+  { label: '日本国', value: LanguageTypes.JAPAN },
+  { label: 'English', value: LanguageTypes.ENGLISH },
+];
+
+export const LanguageSelect: FC = memo(() => {
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/components/LanguageSelect/LanguageSelect.tsx:18]: To keep the consistency with the other files, can you rename the file to `index.tsx`?
And import it like this.

```ts
import { LanguageSelect } from '@/components/LanguageSelect';
```

**Review by cre8ivejp — CHANGES_REQUESTED** [review]

File: ui/web-v2/apps/admin/src/lang/yup/en.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
+export const localEn = {
```
[the remark this answers is not in the document]
Comment (bimalgrg519) [reply ui/web-v2/apps/admin/src/lang/yup/en.ts:1]: Yes. I used google translator for it.

File: ui/web-v2/apps/admin/src/components/Header/index.tsx (hunk 1 above)
Comment (bimalgrg519) [reply ui/web-v2/apps/admin/src/components/Header/index.tsx:3]: got it.

**Review by cre8ivejp — APPROVED** [review]
Awesome!
Thanks!

**Pull request merged**